salt/utils/event.py

```diff
@@ -121,6 +121,14 @@ def get_event(
                            set_event_handler() API. Otherwise, operation
                            will be synchronous.
     """
+    if transport:
+        salt.utils.versions.warn_until(
+            "3006",
```

Comment:
Does this work with the version number? I think it needs to be the Release Name. Also we should warn until the CHLORINE release. We need to give atleast a year's notice and two releases usually hits that mark.

Reply:
I believe so, here is the `warn_until` doc...

```
:param version: The version info or name after which the warning becomes a ``RuntimeError``.
                    For example ``(2019, 2)``, ``3000``, ``Hydrogen`` or an instance of
                    :class:`salt.version.SaltStackVersion` or :class:`salt.version.SaltVersion`.
```

Reply:
I'll change it though because elements are cool.